CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 0.11.0 (2/23/2015)
 ==================
 
+* [#936](https://github.com/intridea/grape/pull/936): Fixed default params processing for optional groups - [@dm1try](https://github.com/dm1try).
```

Comment:
I released 0.11.0 today, so this needs to bump into Next Release, sorry for the hassle.